config/locales/arclight.en.yml

```diff
@@ -7,6 +7,7 @@ en:
       sidebar_section:
         link: 'Open viewer'
     hierarchy:
+      view_all: 'View'
       view_more: 'View more'
```

Comment:
Yeah, I left it in case we were using it elsewhere, but you're right, we aren't so I've removed it and updated this PR.